Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+- In addition, the shop system creates an ID for this transaction.  
+- Actindo links both.   
+
+For Actindo, the authorization transaction serves as a reference to a payment made by a customer.   
+You can use this procedure to display all incoming authorization transactions of a payment service provider. You can check for communication errors between Actindo and the payment service provider that has been caused in a termination of a workflow, for example. In addition, if applicable, you can change the status of a transaction to start continuing further processing or to end it.    
+
+If everything works fine with the posting of a payment, you do not need to monitor the *Authorizations* menu entry. In case of errors you can capture, cancel, or void an authorization transaction manually.   
+> [Info] You can only change the status of an authorization transaction, but not any data.
+
+
+## Check authorization transaction
+An authorization transaction provides numerous payment-related details, which may vary depending on the payment service provider settings. To decide, whether you can manually capture, cancel, or void an authorization transaction, you need to check its details.
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one authorization transaction has been created.
+- It is recommended that you have added the *Status ID* column to your view so that you are able to sort or filter for specific statuses. 
+
+#### Procedure
+
+*Payments > Authorizations > Tab LIST*
+![Authorizations](../../Assets/Screenshots/Payments/Authorizations/LISTAuthorizations.png "[Authorizations]")
+
+1. Check the Authorization transaction status in the *Status* column. The status displays the current stage in the payment process. You can use the status ID prefixed below to filter the list. The following statuses are available: 
+    - **1 - Preparing**  
+        The transaction has been created, but is not yet transferred.
+    - **2 - Unconfirmed**   
+        A refund is reserved but not yet released by an Actindo user. This status is relevant for some functions only, for example, the cancellations and chargebacks.
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider, that the transfer was successfully.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the status information field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void authorization transaction](01_ManageAuthorizations.md#void-authorization-transaction).
+    
+2. Click on the authorization transaction to see the details.   
+    The sub-tabs of the transaction are opened.   
```

Comment:
```suggestion
    The *Authorization "Authorization ID"* view is displayed.   
```
So haben wir immer dieses View genannt (auch z.B. in Fulfillment, Dispatch note "Dispatch note ID"), eigentlich sollte es reintheoretisch ""Autorization ID "Authorization ID"" heißen, aber sieht nicht so schön aus. Hmm, der Grund ist, dieses ID ist eigentlich das vom System vergebenen ID für diese Entität (Authorisierung in diesem Fall). Man kann die Nummer daher über die ID Spalte suchen und so die Entität finden.

Reply:
done